### ISSUE TITLE
siunitx package does not work

### ISSUE THREAD

**Issue by baderas** [sieversMartin/LNI#3]
When adding the line `\usepackage{siunitx}` below `\documentclass[]{lni}` (or somewhere else) in lni-author-template.tex, `pdflatex lni-author-template.tex` fails with:
```
! Missing number, treated as zero.
<to be read again>
                   \let
l.17200 ...nst:Nn \c_sys_year_int  { \tex_year:D }

?
```

Plattform is Windows 10 x64
MiKTeX-pdfTeX 2.9.6000 (1.40.17) (MiKTeX 2.9 64-bit)
Copyright (C) 1982 D. E. Knuth, (C) 1996-2016 Han The Thanh
TeX is a trademark of the American Mathematical Society.
compiled with zlib version 1.2.8; using 1.2.8
compiled with libpng version 1.6.21; using 1.6.21
compiled with poppler version 0.42.0
compiled with jpeg version 8.4

siunitx package has version 2.6q (2016-03-09 19:33:43)

**Comment by sieversMartin** [comment]
Fixed with commit  86ab90b

**Comment by sieversMartin** [comment]
Please note, that (at least) \year now has to be given *after* \begin{document}
